Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseAutonomous.java

```diff
@@ -248,45 +307,192 @@ public void onError(int errorCode) {}
+            sleep(500);
+            driveAutonomous(180, 0.1);
+            //drive slides to stow position
             sleep(100);
+            driveSlidesAutonomous(Constants.SLIDE_LOW);
+            //drive backwards 34.5 inches
+            driveAutonomous(180, 34.7);
+            sleep(100);
+            //turn towards junction
+            turnToAngle(90 + angleOffset);
+            //drive slides up
+            driveSlidesAutonomous(Constants.SLIDE_HIGH - 10);
+            //:TODO EXPERIMENTAL
+            //drive forward with cone centering
+            centerConeStack(robotCameraPipeline, Constants.BREAK_HEIGHTS[i]);
+            telemetry.addData("width", robotCameraPipeline.width);
+            telemetry.update();
+            sleep(100);
+            //lower slides onto junction
+            driveSlidesAutonomous(Constants.SLIDE_HIGH - 300);
             //open the grabber
             driveGrabber(Constants.GRABBER_OPEN_POSITION);
             //wait for cone to drop
             sleep(100);
             //drive slides back up
-            driveSlides(Constants.SLIDE_TOP);
-            sleep(200);
+            driveSlidesAutonomous(Constants.SLIDE_HIGH);
             //drive backwards
-            driveAutonomous(180, 3);
-            sleep(200);
+            driveAutonomous(180, 2);
             //turn back to 0 heading
+        }
+        turnToAngle(0+angleOffset);
+        driveSlidesAutonomous(Constants.STACK_HEIGHTS[0]);
+        //center on cone stack
+        centerConeStack(robotCameraPipeline, 380);
+        //close grabber
+        driveGrabber(Constants.GRABBER_CLOSE_POSITION);
+        //wait for grabber to close
+        sleep(300);
+        driveAutonomous(180, 1);
+        //drive slides to stow position
+        sleep(100);
+        driveSlidesAutonomous(Constants.SLIDE_LOW);
+        //drive backwards 34.5 inches
+        driveAutonomous(180, 33.8);
+        sleep(100);
+        //turn towards junction
+        turnToAngle(90+angleOffset);
+        //drive slides up
+        driveSlidesAutonomous(Constants.SLIDE_HIGH-10);
+        //drive forward with cone centering
+        centerConeStack(robotCameraPipeline, Constants.BREAK_HEIGHTS[0]);
+        telemetry.addData("width", robotCameraPipeline.width);
+        telemetry.update();
+        sleep(100);
+        //lower slides onto junction
+        driveSlidesAutonomous(Constants.SLIDE_HIGH-300);
+        //open the grabber
+        driveGrabber(Constants.GRABBER_OPEN_POSITION);
+        //wait for cone to drop
+        sleep(100);
+        //drive slides back up
+        driveSlidesAutonomous(Constants.SLIDE_HIGH);
+        //drive backwards
+        driveAutonomous(180, 2);
+        //turn back to 0 heading
+    }
+
+    //TODO: INCLUDES DIFFERENT GRABBING BREAK WIDTHS -- IF THE ROBOT DOES NOT SLAM INTO THE WALL, KEEP THESE
+    public void grabFromStackAndDepositOnJunctionPlusConeCenteringPlusSimulSlides(int loopNumber, int angleOffset){
```

Comment:
Can you call `grabFromStackAndDepositOnJunctionPlusConeCentering` in this method then just add the extra stuff on top?